examples/raspberrypi/rp2xxx/src/rp2040_only/i2c_slave.zig

```diff
@@ -0,0 +1,73 @@
+const std = @import("std");
+const microzig = @import("microzig");
+
+const rp2xxx = microzig.hal;
+
+const gpio = rp2xxx.gpio;
+const i2c = rp2xxx.i2c;
+const time = rp2xxx.time;
+const uart = rp2xxx.uart.instance.num(0);
+
+const pin_config = rp2xxx.pins.GlobalConfiguration{
+    .GPIO0 = .{ .name = "gpio0", .function = .UART0_TX },
+    .GPIO4 = .{
+        .name = "sda",
+        .function = .I2C0_SDA,
+        .slew_rate = .slow,
+    },
+    .GPIO5 = .{
```

Comment:
Seems we have them swapped in the i2c bus scan example (I noticed while testing this PP). Could you please fix that?

Reply:
Thank you!